### ISSUE TITLE
how to get rid of "F: Ignoring duplicate content rating"

### ISSUE THREAD

**Issue by btreut** [flatpak/flatpak#4340]
## Linux distribution and version#
endlessOS, 3.9.5
## Flatpak version
Flatpak 1.8.2
## Description of the problem
when I do a `flatpak list` I get several lines of `F: Ignoring duplicate content rating` on stderr, which I do not understand.
## Steps to reproduce
it happens every time I do a `flatpak list`, but I think it is cause by conflicting repos, but I don't have any idea, how I could find out which are conflicting and how to resolve the conflict.

**Comment by hfiguiere** [comment]
it's problably one of the package that has, as the message say, a duplicate content rating in its appstream.

the question is which package.

likelu just noise.

**Comment by btreut** [comment]
Yes  its annoying noise and as far as I understand this here: https://community.endlessos.com/t/flatpak-complains-f-ignoring-duplicate-content-rating-how-to-get-rid-of-this/19580/9 it is caused by some package belonging to EndlessOS

**Comment by hfiguiere** [comment]
closing then

**Comment by razzeee** [comment]
That link seems to be private or not work

**Comment by btreut** [comment]
> That link seems to be private or not work

oops, sorry, it apparently works only when I am logged in at the community ...

So I repeat, what I posted over there:

I cleaned up my installation. When doing a “flatpak update” there were lots of messages about installed obsolete runtimes and applications, which would not get updates anymore. After uninstalling all apps, which were listed as outdated, the messages “F: Ignoring duplicate content rating” vanished also.
